Include/cpython/initconfig.h

```diff
@@ -373,10 +373,11 @@ typedef struct {
                                        module_search_paths_set is equal
                                        to zero. */
 
-    wchar_t *executable;    /* sys.executable */
-    wchar_t *prefix;        /* sys.prefix */
-    wchar_t *base_prefix;   /* sys.base_prefix */
-    wchar_t *exec_prefix;   /* sys.exec_prefix */
+    wchar_t *executable;        /* sys.executable */
+    wchar_t *base_executable;   /* sys.base_executable */
```

Comment:
It's sys._base_executable, no?

Reply:
It is now, yes :) (I started this change by making it public, but I gave up getting it to work on Linux so I'm just making it private again)